pkg/util/log.go

```diff
@@ -0,0 +1,38 @@
+package util
```

Comment:
Rename this package log or logging. util doesn't say anything about what's in it and it's currently only log functions.